buildpack/runtime_components/security.py

```diff
@@ -63,6 +63,10 @@ def create_admin_user(m2ee, is_development_mode):
 
 def get_certificate_authorities():
     config = {}
+    cas64 = os.getenv("CERTIFICATE_AUTHORITIES_BASE64", None)
+    if cas64:
+        certificate_authorities = base64.b64decode(cas64)
```

Comment:
Have you considered trying to base64 decode the original environment variable first? If that passes, you can use that value - if not, you can use the original parsing behaviour.